### ISSUE TITLE
compatibility with linter-spell

### ISSUE THREAD

**Issue by apalkowski** [facebook-atom/atom-ide-ui#110]
### Description
When hovering over several warnings produced by the linter-spell package (with linter-spell-latex) the change options differ and sometimes are incomplete:
![ui-1](https://user-images.githubusercontent.com/5190825/32247218-ef620bf6-be81-11e7-816f-c3a168afb662.png)
![ui-2](https://user-images.githubusercontent.com/5190825/32247240-fa3e89a0-be81-11e7-85f6-8e865f641a6e.png)

### Expected Behavior
The options should contain at least few "Change to..." items and constant "Ignore", "Add to personal dictionary", and "Add to LaTeX dictionary" (in case of LaTeX) options.

### Actual Behavior
it is rather random how the options are displayed. Sometimes there are only "Change to..." options and sometimes there are other.

### Versions
* Atom: 1.21.1 x64
* Client OS: Ubuntu Gnome 16.04
* atom-ide-ui: 0.5.3

### Additional Details
* Installed packages (`apm ls --installed`):
├── Zen@0.18.0
├── atom-beautify@0.30.6
├── atom-ide-ui@0.5.3
├── autocomplete-paths@2.12.1
├── change-case@0.6.5
├── document-outline@1.6.17
├── file-icons@2.1.13
├── ide-python@0.2.3
├── language-latex@1.1.1
├── latextools@0.8.5
├── linter-chktex@1.3.1
├── linter-proselint@3.3.0
├── linter-shellcheck@1.4.6
├── linter-spell@0.15.0
├── linter-spell-latex@0.10.0
├── linter-write-good@0.9.0
├── minimap@4.29.7
├── minimap-find-and-replace@4.5.2
├── project-manager@3.3.5
└── toggler@0.3.0

**Comment by wbinnssmith** [comment]
@hansonw, is this using a similar API as code actions?

**Comment by hansonw** [comment]
Yeah it's converting the linter solutions to code actions. I think we artifically constrained the number of actions to 4 due to UI constraints... combined with JavaScript's unstable sort, that's probably leading to the behavior here. Shouldn't be too hard to patch up.

**Comment by ProgramFan** [comment]
The same problem happens here with atom 1.23.3. Is it possible to add a scrollbar in the UI to remove the 4-action constraint?
